scripts/gitlab/build-windows.sh

```diff
@@ -31,7 +31,7 @@ echo "_____ Calculating checksums _____"
 for binary in $(ls)
 do
   rhash --sha256 $binary -o $binary.sha256
-  parity.exe tools hash $binary > $binary.sha3
+  ./parity.exe tools hash $binary > $binary.sha3
```

Comment:
`parity.exe` is for sure in the correct directory, but I guess it has to be invoked in a unix-like fashion